client/index.js

```diff
@@ -20,7 +20,7 @@ const {
 
 const Component = evalScript(component).default
 const ErrorComponent = evalScript(errorComponent).default
-let lastAppProps
```

Comment:
When the props became `null` our code tries to read the props from `lastAppProps`.

Then, For the very first doRender call, it tries to check few items from this `lastAppProps`. Unfortunately, it's a null and we'll get the mentioned problem.